common/app/common/models/TitleFormFieldBean.java

```diff
@@ -0,0 +1,48 @@
+package common.models;
+
+import common.contexts.UserContext;
+import common.template.i18n.I18nIdentifier;
+import common.template.i18n.I18nResolver;
+import io.sphere.sdk.models.Base;
+import play.Configuration;
+
+import javax.annotation.Nullable;
+import java.util.List;
+
+import static java.util.Collections.emptyList;
+import static java.util.stream.Collectors.toList;
+
+public class TitleFormFieldBean extends Base {
+
+    private static final String CONFIG_TITLE_OPTIONS = "form.titles";
+
+    private List<SelectableBean> list;
+
+    public TitleFormFieldBean() {
+    }
+
+    public TitleFormFieldBean(@Nullable final String selectedTitle, final UserContext userContext,
+                              final I18nResolver i18nResolver, final Configuration configuration) {
```

Comment:
Yes, I also wanted to remove the Configuration from there, also because it has to be loaded every single time. But wanted to aimed for that later, so thanks for the inspiration, I will try to change it once I have clear the best way:
https://github.com/sphereio/commercetools-sunrise-java/issues/379